website/content/docs/api-gateway/tech-specs.mdx

```diff
@@ -17,12 +17,8 @@ Verify that your environment meets the following requirements prior to using Con
 
 Your datacenter must meet the following requirements prior to configuring the Consul API Gateway:
 
-- Kubernetes 1.21+
-  - Kubernetes 1.24 is not supported at this time.
-- `kubectl` 1.21+
-- Consul 1.11.2+
-- HashiCorp Consul Helm chart 0.45.0+
-- Consul Service Mesh must be deployed on the Kubernetes cluster that API Gateway is deployed on.
+- Versions of Kubernetes, `kubectl`, Consul and the Consul Helm chart are determined by the Consul API Gateway version deployed. Refer to [Release Notes](/docs/release-notes) for details.
```

Comment:
I would have loved to link to `/docs/release-notes/api-gateway`; however, that's not an actual page and would require specifying a specific release in the URL

Reply:
@nathancoleman I think we need to keep this information here. The release notes should provide a history of the product changes, but the core docs should always contain requirements/technical specs. This is consistent with how other Consul add-ons document requirements.

Reply:
This makes sense. Will revert the most recent commit.

FYI @Jeff-Apple